jisantuc/panrec
# Require an output directory in cli

Overview
-----

This PR requires an output directory from the cli in order to convert anything. It also does a better job handling whether the input has a trailing slash. Thanks, `System.FilePath.Posix`!

Testing
-----

- `stack install`
- copy the whole raster foundry datamodel into, e.g., `./data` from the repo root
- `mkdir converted`
- `panrec -i scala -o ts-interface data converted`
- do any combination of leading `./`s and trailing `/`s you want, and everything should still be fine
- try the same with a specific file from inside `data`

Closes #27
